src/components/Signup/Signup.tsx

```diff
@@ -0,0 +1,26 @@
+import { useState } from "react";
+import ConfirmForm from "./ConfirmForm";
+import SignupForm from "./SignupForm";
+import Success from "./Success";
+import { Donor, SignupState } from "./types";
+
+type Props = { classes?: string; donor: Donor };
+
+export default function Signup({ classes = "", donor }: Props) {
```

Comment:
I assume it was created to be shared between `DonateFiatThanks` and the Sign up page. This can be done, but need to clarify some things first.

The problem is that the sign up page has too many different components that are interspersed between the ones in this components:
- The "You are signing up as" section
- first name, last name, email fields

Basically the only common fields are: 1. password field & 2. sign up button

We could handle this like the following:
1. if `donor` is defined, treat this as a subcomponent of DonateFiatThanks
2. otherwise, treat this as a subcomponent of the actual Sign up page with all the other fields

In the former case we would hide all the unnecessary fields/components, while in the latter we would display them. This would result in many `{!!donor && <SomeFieldOrComponent />}` instances.

If this is ok with you, we can leave this as-is. If not, we should move this component to be a part of the `pages/DonateFiatThanks/` folder and I'll create a completely new Signup page with some copy/pasted from here.